src/main/kotlin/com/hibob/academy/resource/petsResource.kt

```diff
@@ -0,0 +1,49 @@
+package com.hibob.academy.resource
+
+import com.hibob.academy.types.Pet
+import jakarta.ws.rs.*
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.RequestBody
+import java.util.*
+
+@Controller
+@Path("/api/noy/pets")
+@Produces(MediaType.APPLICATION_JSON)
+class PetsResource {
+
+// create
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/newPet")
```

Comment:
Usually for create we don't always need to define a name to the route. 
So because we're doing REST protocol, the create is understood automatically when you defined the resource. 
So a POST request to `/api/noy/pets` would also be a create.
A GET request to `/api/noy/pets` would give you a list of all pets.